src/Solver/Interoperability.cpp

```diff
@@ -569,7 +569,8 @@ void seissol::Interoperability::initializeModel(  char*   materialFileName,
 
   //first initialize the (visco-)elastic part
   auto nElements = seissol::SeisSol::main.meshReader().getElements().size();
-  seissol::initializers::ElementBarycentreGenerator queryGen(seissol::SeisSol::main.meshReader());
+  //seissol::initializers::ElementBarycentreGenerator queryGen(seissol::SeisSol::main.meshReader());
+  seissol::initializers::ElementAverageGenerator queryGen(seissol::SeisSol::main.meshReader());
```

Comment:
Please implement a check, such that we use a BarycentreGenerator for poroelastic and anisotropic materials and an AverageGenerator for (visco)elastic materials.

Reply:
Added check in Interoperability.cpp